src/mono/wasm/build/WasmApp.Common.targets

```diff
@@ -620,7 +620,7 @@
     </PropertyGroup>
 
     <Error Condition="'$(RunAOTCompilation)' == 'true' and '$(PublishTrimmed)' != 'true'"
-           Text="AOT is not supported without IL trimming (PublishTrimmed=true required)." />
+           Text="AOT is not supported without IL trimming (EnableAggressiveTrimming=true or PublishTrimmed=true required)." />
```

Comment:
Is EnableAggressiveTrimming a user-facing thing? Isn't it just runtime repo thing?